core/arch/arm/plat-imx/conf.mk

```diff
@@ -425,5 +425,8 @@ CFG_IMX_CAAM ?= y
 endif
 endif
 
+CFG_IMX_I2C ?= n
+CFG_IMX_I2C_CLK_RATE ?= 24000000
```

Comment:
Good thing to think about the others, but if you don't make use of it personally, I would like to see the clock rate defined in the header file like it was before.

Reply:
I disagree 100% with you on this but sure will remove and hack the driver to a fix clock rate.